vtexdocs/RapiDoc
# Feat/column layout

#### What is the purpose of this pull request?

Implement the two column layout in the API reference pages and the communication between the RapiDoc sections.

#### What problem is this solving?

It makes easier to test the API since the components will be divided into two columns. And when an input is changed the new value is showed instantly in the code part of the RapiDoc section.

#### How should this be manually tested?

Go to [the preview page](https://deploy-preview-108--elated-hoover-5c29bf.netlify.app/) and check if the layout in the API Reference pages is displayed correctly in all resolutions and if when an input is changed the curl is updated.

#### Screenshots or example usage

Further information, screenshots and videos can be found [here](https://www.notion.so/vtexhandbook/Implementar-layout-de-duas-colunas-na-p-gina-de-API-reference-12f10119422a4b8d8d2d86b7a3645b38) and [here](https://www.notion.so/vtexhandbook/Comunicar-estado-das-se-es-OpenAPI-rapidoc-e51d2283131247e0a037abab5d605f1b).

![Captura de tela de 2022-12-20 10-49-32](https://user-images.githubusercontent.com/62757720/208682180-6bfc7343-f661-43dd-8c37-d031253bb335.png)

#### Types of changes
